backend/src/routes/projectRouter.ts

```diff
@@ -205,6 +205,17 @@ projects.get("/v1/projects", async (req, res, next) => {
   res.json(projects);
 });
 
+// Returns the session user's projects
+projects.get("/v1/user_projects", async (req: Request, res, next) => {
```

Comment:
I think we should go with a more RESTful endpoint path, with a semantic description of "member collection lookup with user ID filter"

Like: `/api/v1/members?user={userId}` with the projects and users populated

example response:
```
[
  {
    "id": "member123",
    "user": {
      ... user info from ID
    },
    "project": {
      "id": "project1",
      "name": "Project 1",
      ...
    },
    "roleName": "owner"
  },
  {
    "id": "member124",
    "user": {
      ... user info from ID
    },
    "project": {
      "id": "project2",
      "name": "Project 2",
      ...
    },
    "roleName": "developer"
  },
  ...
]
```